GltfExport.cs

```diff
@@ -64,93 +68,103 @@ protected override void SolveInstance(IGH_DataAccess DA)
                 Message = "Scene";
             }
 
-            FileGltfWriteOptions optionsGltf =
+            FileGltfWriteOptions? optionsGltf =
                 options != null
-                && options! is FileGltfWriteOptions ?
+                && options is FileGltfWriteOptions ?
                 options as FileGltfWriteOptions : DefaultOptions();
 
-            if (!export) return;
-            string res = "";
+            if (!export || _executing)
```

Comment:
# Flow - Goto statements
 - generally discouraged bc making it harder to follow and maintain.
 - might be necessary for thread management in this context, but we should consider alternative approaches:
    - early returns
    - breaking logic down into smaller methods
  - to improve readability and future modifiability.